src/payment/checkout/Checkout.jsx

```diff
@@ -273,6 +274,7 @@ class Checkout extends React.Component {
               isBulkOrder={isBulkOrder}
```

Comment:
This is a comment for L273, we'll need to add a new `handleSubmitStripeButtonClick` function for event tracking for subs, and pass that on as props, if the assumption is that we'll need a different event name than regular payments. Let's PL today.

Reply:
noted.